tap_salesforce/salesforce/__init__.py

```diff
@@ -168,6 +168,9 @@ def field_to_property_schema(field, mdata, source_type):  # pylint:disable=too-m
         property_schema['type'] = ["number", "string", "null"]
     elif sf_type in NUMBER_TYPES:
         property_schema['type'] = "number"
+    elif sf_type == "percent":
+        # percent type field in SF Object returns numeric value without %, but SF Report returns numeric value with %
+        property_schema['type'] = "string" if is_report else "number"
```

Comment:
- initially, percent type column was imported as string for both reports and objects.
- after switching to target-s3-parquet, percent type column was imported as number for both reports and objects. However, returned data from Salesforce included '%' for reports, but not for objects causing percent type column to be empty for reports.
- As a fix, we will be importing percent type as string for reports, and keep the same behaviour for objects. 
- Other options like removing '%' or cleaning the data from target was disregarded due to potential performance issue.